Automattic/wp-calypso
# Remove importer flows from the signup framework

#### Proposed Changes

* Remove importer flows configuration
* Remove importer flows entry components
* Update intent screen to redirect to the stepper framework flow

#### Testing Instructions

* Go to `/start/setup-site/intent?siteSlug={SLUG}`
* Select `Import your site content` option
* Check if redirects to the `/setup/import` flow

Related to #64065
